lib/rmt.toit

```diff
@@ -50,7 +50,11 @@ At the lower level, a signal consists of 16 bits: 15 bits for the period and 1
   to be divisible by 4.
 */
 class Signals:
-  /** An empty signals instance. */
+  /**
+  An empty signals instance.
+
+  Deprecated. Signals should not be empty.
```

Comment:
Didn't you just add this? I guess we didn't release anything with `Signals.ZERO` support in it.

Reply:
It was already there, but in a different place (a bit lower).

https://libs.toit.io/rmt/class-Signals#ZERO(0%2C0%2C0%2C)